### DISCUSSION TITLE
[feature] Simplified Chinese Web and New Version

### DISCUSSION THREAD

**Discussion by PolarisGitYHNL** [immich-app/immich#10058] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

We have already completed most of the translation work for Simplified Chinese. When will IMMICH update the next version and merge the Chinese translations so that we can gather translation opinions and correct any insufficient or incorrect translations in the Chinese IMMICH communication community.
Thank you very much for the work of the developers. We, as Simplified Chinese users, also hope to contribute to immich.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
The next release will be out this week
